lib/orangetheses.rb

```diff
@@ -1,7 +1,7 @@
 require 'orangetheses/version'
 
 module Orangetheses
-  SET = 'hdl_88435_dsp019c67wm88m'
+  SET = 'com_88435_dsp019c67wm88m'
```

Comment:
yeah it did: http://dataspace.princeton.edu/oai/request?verb=ListIdentifiers&metadataPrefix=oai_dc&set=com_88435_dsp019c67wm88m